metricbeat/module/aerospike/aerospike.go

```diff
@@ -47,6 +57,24 @@ func ParseClientPolicy(config Config) (*as.ClientPolicy, error) {
 		clientPolicy.TlsConfig = tlsconfig.ToConfig()
 	}
 
+	if len(config.User) > 0 && len(config.Password) > 0 {
```

Comment:
[NIT]

As you're checking for empty string, it's more readable do it explicitly:

```suggestion
	if config.User != "" && config.Password != "" {
```